config/config.go

```diff
@@ -6,9 +6,9 @@ import "github.com/kelseyhightower/envconfig"
 type Specification struct {
 	Port           string `default:"8080" envconfig:"PORT"`
 	Polling        int    `default:"43200" envconfig:"POLL_INTERVAL_SEC"` // 43200 seconds = 12 hours
-	VersionsAPIURL string `envconfig:"VERSIONS_API_URL"`
-	DoctorAPIURL   string `envconfig:"DOCTOR_API_URL"`
-	APIVersion     string `envconfig:"API_VERSION"`
+	VersionsAPIURL string `envconfig:"VERSIONS_API_URL" default:"https://versions-staging.deis.com"`
```

Comment:
:+1: I just documented this as the defaults, but felt it was odd that the chart defined these envvars